src/include/mallocMC/reservePoolPolicies/CudaSetLimits_impl.hpp

```diff
@@ -27,6 +27,8 @@
 
 #pragma once
 
+#ifdef ALPAKA_ACC_GPU_CUDA_ENABLED
```

Comment:
I also disabled this for non-CUDA targets. Is there any way I can implement this for CPU targets?